src/components/Text/Link.js

```diff
@@ -1,38 +1,112 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-import { colors } from "../../theme";
+import Base from "./Base.styles";
 import StyledText from "./StyledText";
+import colors from "../../theme/colors";
 import getRelByTarget from "../../utils/link";
+import { getFontColor } from "../../utils/typography";
 
 export const LinkTitle = StyledText.withComponent(`a`).extend`
   color: ${({ color }) => color || colors.azure.base};
   text-decoration: none;
 `;
 
-const Link = ({ href, children, ...props }) => {
+const LinkBase = Base.withComponent("a").extend`
+  text-decoration: none;
+  transition: color 0.3s ease;
+  &:focus,
+  &:active,
+  &:visited,
+  &:hover {
+    color: ${props => getFontColor(props)};
+  }
+
+  &:hover {
+    color: ${colors.azure.hover}
+  }
+`;
+
+const Link = ({ href, children, weight, size, responsiveSize, ...props }) => {
   const { target, rel } = props;
   const validatedRel = getRelByTarget(target, rel);
 
   return (
-    <LinkTitle href={href} rel={validatedRel} {...props}>
+    <LinkBase
+      {...props}
+      size={{
+        small: responsiveSize.small || size,
+        medium: responsiveSize.medium || responsiveSize.small || size,
+        large:
+          responsiveSize.large ||
+          responsiveSize.medium ||
+          responsiveSize.small ||
+          size
+      }}
+      primary
+      variant="accent"
+      accent="azure"
```

Comment:
Is there a separate LN variant that we should be supporting here? Unsure if styles have been defined for an LN aurora link variant.

Reply:
I'm thinking this will change based on the currently open MR for colors. That MR bring in LN styles.